h/browser/chrome/lib/tab-state.js

```diff
@@ -1,96 +1,94 @@
-(function (h) {
-  'use strict';
-
-  var states = {
-    ACTIVE:   'active',
-    INACTIVE: 'inactive',
-    ERRORED:  'errored',
-  };
-
-  /* Manages the state of the browser action button to ensure that it displays
-   * correctly for the currently active tab. An onchange callback will be
-   * called when the extension changes state. This will be provided
-   * with the tabId, the current and previous states.
+//'use strict';
```

Comment:
No, that's a mistake. In future I can get the build tooling to add `'use strict'` automatically rather than having to add it to each file manually.